.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thanks for opening a Pull Request!
+
+Here's a handy checklist to ensure your PR goes smoothly.
+
+- [] Ensure you've signed Google's
+[Contributor License Agreement](https://opensource.google.com/docs/cla/)
+- [] Ensure the code conforms to Google's
+[python style](https://google.github.io/styleguide/pyguide.html).
+- [] Ensure that your PR includes increased unit-test coverage (if applicable).
```

Comment:
Instead of ```increased```, how about ```applicable``` or do not decrease the test coverage.
```Ensure that your PR includes applicable unit-test coverage (or at least, do not decrease coverage).```